diff --git a/src/utils.c b/src/utils.c
index ee5073d..00bebac 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -365,19 +365,19 @@ parse_iso_time_tz (const char *text_time, const char *fallback_tz)
                             ? secs_str : ":00",
                            offset_str ? offset_str : "");
 
-      if (strptime_with_reset ((char*) cleaned_text_time, "%FT%T%z", &tm))
+      if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time with numeric offset (e.g. 2020-06-01T01:02:03+04:30) */
           tm.tm_sec = tm.tm_sec - tm.tm_gmtoff;
           tm.tm_gmtoff = 0;
           epoch_time = mktime_with_tz (&tm, "UTC");
         }
-      else if (strptime_with_reset ((char*) cleaned_text_time, "%FT%TZ", &tm))
+      else if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time with "Z" for UTC timezone (e.g. 2020-06-01T01:02:03Z) */
           epoch_time = mktime_with_tz (&tm, "UTC");
         }
-      else if (strptime_with_reset ((char*) cleaned_text_time, "%FT%T", &tm))
+      else if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time without timezone suffix (e.g. 2020-06-01T01:02:03) */
           epoch_time = mktime_with_tz (&tm, fallback_tz ? fallback_tz : "UTC");
