diff --git a/src/manage.c b/src/manage.c
index 7e39631..3c84cee 100644
--- a/src/manage.c
+++ b/src/manage.c
@@ -5788,7 +5788,7 @@ manage_scap_update_time ()
   if (strptime (content, "%Y%m%d%H%M", &update_time))
     {
       static char time_string[100];
-      strftime (time_string, 99, "%FT%T.000%z", &update_time);
+      strftime (time_string, 99, "%Y-%m-%dT.000", &update_time);
       return time_string;
     }
   return "";
diff --git a/src/manage_sql.c b/src/manage_sql.c
index 14d9e35..88d96c3 100644
--- a/src/manage_sql.c
+++ b/src/manage_sql.c
@@ -26202,7 +26202,7 @@ host_summary_append (GString *host_summary_buffer, const char *host,
           struct tm start_tm;
 
           memset (&start_tm, 0, sizeof (struct tm));
-          if (strptime (start_iso, "%FT%H:%M:%S", &start_tm) == NULL)
+          if (strptime (start_iso, "%Y-%m-%dT%H:%M:%S", &start_tm) == NULL)
             {
               g_warning ("%s: Failed to parse start", __func__);
               return;
@@ -26222,7 +26222,7 @@ host_summary_append (GString *host_summary_buffer, const char *host,
           struct tm end_tm;
 
           memset (&end_tm, 0, sizeof (struct tm));
-          if (strptime (end_iso, "%FT%H:%M:%S", &end_tm) == NULL)
+          if (strptime (end_iso, "%Y-%m-%dT%H:%M:%S", &end_tm) == NULL)
             {
               g_warning ("%s: Failed to parse end", __func__);
               return;
diff --git a/src/utils.c b/src/utils.c
index ee5073d..b311966 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -365,19 +365,19 @@ parse_iso_time_tz (const char *text_time, const char *fallback_tz)
                             ? secs_str : ":00",
                            offset_str ? offset_str : "");
 
-      if (strptime_with_reset ((char*) cleaned_text_time, "%FT%T%z", &tm))
+      if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time with numeric offset (e.g. 2020-06-01T01:02:03+04:30) */
           tm.tm_sec = tm.tm_sec - tm.tm_gmtoff;
           tm.tm_gmtoff = 0;
           epoch_time = mktime_with_tz (&tm, "UTC");
         }
-      else if (strptime_with_reset ((char*) cleaned_text_time, "%FT%TZ", &tm))
+      else if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time with "Z" for UTC timezone (e.g. 2020-06-01T01:02:03Z) */
           epoch_time = mktime_with_tz (&tm, "UTC");
         }
-      else if (strptime_with_reset ((char*) cleaned_text_time, "%FT%T", &tm))
+      else if (strptime_with_reset ((char*) cleaned_text_time, "%Y-%m-%dT%T", &tm))
         {
           /* ISO time without timezone suffix (e.g. 2020-06-01T01:02:03) */
           epoch_time = mktime_with_tz (&tm, fallback_tz ? fallback_tz : "UTC");
@@ -427,7 +427,7 @@ iso_time_internal (time_t *epoch_time, const char **abbrev)
   if (timezone == 0)
 #endif
     {
-      if (strftime (time_string, 98, "%FT%TZ", &tm) == 0)
+      if (strftime (time_string, 98, "%Y-%m-%dT%T", &tm) == 0)
         return NULL;
 
       if (abbrev)
@@ -437,7 +437,7 @@ iso_time_internal (time_t *epoch_time, const char **abbrev)
     {
       int len;
 
-      if (strftime (time_string, 98, "%FT%T%z", &tm) == 0)
+      if (strftime (time_string, 98, "%Y-%m-%dT%T", &tm) == 0)
         return NULL;
 
       /* Insert the ISO 8601 colon by hand. */
