diff -Naur origin/gsa/src/gmp/gmpsettings.js patch/gsa/src/gmp/gmpsettings.js
--- origin/gsa/src/gmp/gmpsettings.js	2021-04-16 11:27:51.000000000 +0200
+++ patch/gsa/src/gmp/gmpsettings.js	2021-06-28 10:08:42.787815562 +0200
@@ -26,7 +26,7 @@
   'https://docs.greenbone.net/API/GMP/gmp-20.08.html';
 export const DEFAULT_REPORT_RESULTS_THRESHOLD = 25000;
 export const DEFAULT_LOG_LEVEL = 'warn';
-export const DEFAULT_TIMEOUT = 300000; // 5 minutes
+export const DEFAULT_TIMEOUT = 90000000; // 5 minutes
 
 const set = (storage, name, value) => {
   if (isDefined(value)) {
diff -Naur origin/gsa/src/gmp/__tests__/gmpsettings.js patch/gsa/src/gmp/__tests__/gmpsettings.js
--- origin/gsa/src/gmp/__tests__/gmpsettings.js	2021-04-16 11:27:51.000000000 +0200
+++ patch/gsa/src/gmp/__tests__/gmpsettings.js	2021-06-28 10:08:27.144046612 +0200
@@ -94,7 +94,7 @@
       reloadIntervalInactive: 60,
       reportResultsThreshold: 10000,
       token: 'atoken',
-      timeout: 30000,
+      timeout: 9000000,
       timezone: 'cet',
       username: 'foo',
       vendorVersion: 'foo',
@@ -118,7 +118,7 @@
     expect(settings.reloadIntervalInactive).toEqual(60);
     expect(settings.reportResultsThreshold).toEqual(10000);
     expect(settings.token).toBeUndefined();
-    expect(settings.timeout).toEqual(30000);
+    expect(settings.timeout).toEqual(9000000);
     expect(settings.timezone).toBeUndefined();
     expect(settings.username).toBeUndefined();
     expect(settings.vendorVersion).toEqual('foo');
@@ -214,7 +214,7 @@
       reloadIntervalInactive: 60,
       reportResultsThreshold: 10000,
       token: 'atoken',
-      timeout: 30000,
+      timeout: 9000000,
       timezone: 'cet',
       username: 'foo',
       vendorVersion: 'bar',
@@ -234,7 +234,7 @@
     expect(settings.reloadIntervalInactive).toEqual(60);
     expect(settings.reportResultsThreshold).toEqual(10000);
     expect(settings.token).toEqual('btoken');
-    expect(settings.timeout).toEqual(30000);
+    expect(settings.timeout).toEqual(9000000);
     expect(settings.timezone).toEqual('cest');
     expect(settings.username).toEqual('bar');
     expect(settings.vendorVersion).toEqual('bar');
@@ -306,7 +306,7 @@
       protocolDocUrl: 'http://protocol',
       reloadInterval: 10,
       token: 'atoken',
-      timeout: 30000,
+      timeout: 9000000,
       timezone: 'cet',
       username: 'foo',
       vendorVersion: 'foobar',
