diff -ruN aports2/community/py3-gvm/APKBUILD aports/community/py3-gvm/APKBUILD
--- aports2/community/py3-gvm/APKBUILD	2021-07-01 15:30:00.528710238 +0200
+++ aports/community/py3-gvm/APKBUILD	2021-06-28 21:19:04.164047714 +0200
@@ -3,7 +3,7 @@
 pkgname=py3-gvm
 _pkgname=${pkgname/py3/python}
 pkgver=21.6.0
-pkgrel=10
+pkgrel=0
 pkgdesc="Greenbone Vulnerability Management Python Library "
 url="https://github.com/greenbone/python-gvm"
 arch="noarch"
